src/clusterfuzz/_internal/system/environment.py

```diff
@@ -786,6 +787,14 @@ def platform():
   raise ValueError('Unsupported platform "%s".' % sys.platform)
 
 
+def platform_vesion():
```

Comment:
Is this needed? Also it's spelled wrong.

Reply:
Chrome folks asked for it, as per design doc. I think it is a good idea to have it, so we can, for instance, figure out what glibc to expect in every environment without having to ask around or read internal config/code